src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -4,13 +4,38 @@
     <Identity Id="0b5e8ddb-f12d-4131-a71d-77acc26a798f" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Language Services</DisplayName>
     <Description>C# and VB.NET language services for Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\Language Services</ShortcutPath>
```

Comment:
It is probably ideal to move FSharp under ManagedLanguages too (since F#, VB and C# are all managed languages) :) But this is not urgent and can wait until later...